WireUI/Sources/WireSidebarUI/Views/SidebarView.swift

```diff
@@ -136,7 +136,35 @@ public struct SidebarView<AccountImageView>: View where AccountImageView: View {
         }
     }
 
-    private func menuItem(_ menuItem: SidebarMenuItem) -> some View {
+    private func nonselectableMenuItem(_ menuItem: SidebarNonselectableMenuItem) -> some View {
+        let text: Text
+        let icon: String
+        let isLink: Bool
+        let action: () -> Void
+        switch menuItem {
+        case .connect:
+            text = Text("sidebar.contacts.connect.title", bundle: .module)
```

Comment:
sorry if I asked this before, but why we can't use swiftgen strings

Reply:
I had issues setting up SwiftGen for our packages. I created a ticket to fix it later: https://wearezeta.atlassian.net/browse/WPB-11610